quicktest.go

```diff
@@ -150,12 +151,26 @@ func (c *C) Assert(got interface{}, checker Checker, args ...interface{}) bool {
 	return c.check(c.TB.Fatal, checker, got, args)
 }
 
+var (
```

Comment:
I wonder if it could be useful at this point to move Run in its own file, run.go for instance.

Reply:
I don't think that's worth it currently.